src/controllers/userController.js

```diff
@@ -139,6 +154,85 @@ class Authentication {
     })(req, res);
   }
 
+  /**
+   * @description admins delete a user
+   * @param {object} req
+   * @param {object} res
+   * @returns {obkject} success message
+   */
+  static async deleteUser(req, res) {
+    try {
+      const name = req.params.username;
+      const userRole = req.userData.role;
+      const user = await User.findOne({ where: { userName: name } });
+      if (!user) {
+        return res.status(404).json({
+          error: `User with username ${name} not found`
+        });
+      }
+      if (userRole === 'user' || ((userRole !== 'admin' || userRole !== 'super-admin')
```

Comment:
I think you can use shorthand here.
if (userRole === 'user' || ((userRole !== ('admin' || 'super-admin'))............

Reply:
Yeah! fixed! Thanx...